experimenter/experimenter/targeting/constants.py

```diff
@@ -2005,6 +2005,26 @@ def __post_init__(self):
     application_choice_names=(Application.DESKTOP.name,),
 )
 
+DEBUG_INELIGIBLE_FOR_DEFAULT_PDF_HANDLER = NimbusTargetingConfig(
+    name="DEBUG Ineligible for default PDF handler",
+    slug="debug_ineligible_for_default_pdf_handler",
+    description=(
+        "Targeting users that are ineligible to set Firefox as the default PDF "
+        "handler or have already done so. This omits safe navigators so bugs "
+        "in the targeting getters can be debugged."
+    ),
+    targeting=(
+        "(!os.isWindows || os.windowsVersion < 10) "
```

Comment:
@aminomancer defaultPDFHandler on non-windows is easy to replicate. It will help if we can land 

`ELIGIBLE_FOR_DEFAULT_PDF_HANDLER` thats known to return `defaultPDFHandler` value duting early startup

Reply:
Thinking more, I believe JEXL will still evaluate rest of the expression on windows even if `(!os.isWindows || os.windowsVersion < 10) ` evaluates `false` and should see `defaultPDFHandler` exception to debug. 

But if we have to land one targeting for debugging will encourage `DEBUG_ELIGIBLE_FOR_DEFAULT_PDF_HANDLER` which should evaluate `true` with no exception irrespective of JEXL order

Reply:
I think defaultPDFHandler only works on windows. I figured I'd just add the one that Marius used for the bug last time since at least we know for sure it throws